server/pbenchinacan/deploy.example

```diff
@@ -0,0 +1,21 @@
+# source this file to construct a standalone test Pbench Server pod with
```

Comment:
This is what `jenkins/runlocal` is for.

Reply:
That's both more and less than I want, though it's useful in some cases.

I'm not trying to (just) run functional tests against a transient local server, and I'm usually not interested in building a new RPM or container.

I want to stand up a development pod that I can access externally, usually on my RDU2 VM, from `main` or from a PR tag. This is exactly the simplistic and direct `source pod` script we were using during the hackathon, but expanded for all the additional environment variables that have been added.

I just figured it would be useful to have a reference in the repo rather than keeping my own off to the side.

Reply:
Okay, then, check our #3154.

Reply:
I think that #3154 is good, and, with that, we won't need this.

Reply:
I expect that *I'll* continue to use it, but apparently nobody wants it in the repo so I'll just suffer more or less in silence.

Reply:
I added the script in PR #3154 but made is used by the existing scripts so it is "in the fold."

Reply:
> I added the script in PR #3154 but made is used by the existing scripts so it is "in the fold."

And I'll probably shift to using that the next time the set of environment variables changes incompatibly; however since I'd end up wrapping it in my own script to change defaults there's not much value in adopting it right now. The refactoring is still good, though.